changelog/12086.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:improvement
```

Comment:
Docs don't get CL entries, they're really about the product/binary that we ship.

Reply:
Thank you, I'll remove!